### PR TITLE
Updates for Read the Docs page

### PR DESCRIPTION
**Pull request by holmesclaire**
Lost of testing to see how the page turned out. Most notably:
- fixing the navigation of different sections
- displaying the images and annotations in table 
- having the python file run on every build of the read the docs page

For Dora's page based off the base MNL repo: change settings to rebuild page every time there is a new commit to the repo. Every rebuild will run the python file that creates the table with all the images and annotations and shows the timestamp of when that python file ran (to make sure it is updating).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # nsd_hed_labels
 This repository contains [Hierarchical Event Descriptors (HED)](https://www.hedtags.org/) labels of 1015 images from the [Natural Scenes Dataset (NSD)](https://naturalscenesdataset.org/) images. This is work in progress...
 
+## Read the Docs 
+[https://nsd-hed-labels-rtd-test.readthedocs.io/en/latest/](https://nsd-hed-labels-rtd-test.readthedocs.io/en/latest/)
```
Comment (tpatpa) [inline README.md]: Change to MNL reathedocs version: https://nsd-hed-labels.readthedocs.io/en/latest/

File: docs/generate_docs.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    import pandas
+    import datetime
+    from mdutils.mdutils import MdUtils
+
+    # load tsv of image identifiers and annotations
+    shared1000 = pandas.read_table('shared1000_HED.tsv')
+
+    # start markdown file
+    mdFile = MdUtils(file_name='nsd_images_annotations', title='Images and Annotations', author= 'Claire Holmes')
+    mdFile.new_line(str(datetime.datetime.utcnow()) + '\n')
```
Comment (tpatpa) [inline docs/generate_docs.py]: Few suggestions here:
1. Add text before timestamp, perhaps 'Last update:'.
2. Format timestamp. For example: `datetime.datetime.utcnow().strftime('%B %d %Y - %H:%M:%S')`
3. Move to end of text before table.

File: docs/generate_docs.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    import pandas
+    import datetime
+    from mdutils.mdutils import MdUtils
+
+    # load tsv of image identifiers and annotations
+    shared1000 = pandas.read_table('shared1000_HED.tsv')
+
+    # start markdown file
+    mdFile = MdUtils(file_name='nsd_images_annotations', title='Images and Annotations', author= 'Claire Holmes')
```
Comment (tpatpa) [inline docs/generate_docs.py]: Change title to 'Images and HED Annotations'

File: docs/generate_docs.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    mdFile.new_line(str(datetime.datetime.utcnow()) + '\n')
+
+    # set up headers for table
+    mdFile.new_line('The images on this webpage have been downsampled from [425, 425] to [200, 200] for upload to GitHub and this webpage.\n')
+    mdFile.new_line('**Only the first 4 images have been uploaded to GitHub and displayed here as an example. All images will be uploaded pending input from collaborators. All links to COCO images should be working. (note: actual NSD images may be a cropped version of the COCO image. The cropped version is the one displayed here and on GitHub)**\n')
```
Comment (tpatpa) [inline docs/generate_docs.py]: Move the crop note to a new line.

File: docs/index.md [hunk 5]
```diff
@@ -5,10 +5,10 @@ This repository contains [Hierarchical Event Descriptors (HED)](https://www.hedt
 1000 images and their annotations are included in the shared1000_HED.tsv. An additional 15 images and their annotations are included in notShown_HED.tsv. The column titled nsd_id corresponds to the ID of the image as seen in the NSD. The cocoId column corresponds to the indentifier in [COCO](https://cocodataset.org/#home) (note: the images used for NSD and these annotations may be a cropped version of the COCO image). Images are shown to participants with a red fixation dot in the center of the image.
 
 ## HED tagging
-All tags are HED validated and are from the standard schema. Both short-form and long-form annotations are included.
+All tags are HED validated and are from the standard schema. Both short-form and long-form annotations are included in the tsv files uploaded to the [GitHub repository](https://github.com/MultimodalNeuroimagingLab/nsd_hed_labels).
```
Comment (tpatpa) [inline docs/index.md]: Add a link to the standard schema (https://github.com/hed-standard/hed-schemas/blob/main/standard_schema/hedwiki/HED8.2.0.mediawiki) and version used (8.2.0).

File: docs/index.md [hunk 6]
```diff
@@ -24,24 +24,24 @@ All tags are HED validated and are from the standard schema. Both short-form and
...
-**Faces**  
+#### **Faces**  
 The Face tag is always accompanied by an Away-from tag (side-profile) or a Towards tag (full face) as measures of how prominent a face is.
 
-**Agents**  
+#### **Agents**  
 Agent tags (Human-agent and Animal-agent) are used in addition to their Item tags (Human and Animal) when there is evidence of an active motion within the image. Good examples of this are legs in an active walking/running motion, water spray from a surfboard to show it is moving through the water, and snow spray from skis to show movement through the snow.
 
-# Contributors (alphabetically)
+## Contributors (alphabetically)
```
Comment (tpatpa) [inline docs/index.md]: Remove '(alphabetically)' from header

File: docs/index.md [hunk 7]
```diff
@@ -24,24 +24,24 @@ All tags are HED validated and are from the standard schema. Both short-form and
...
 If you make substantial changes to this repository, please also feel free to add your name as a contributor
```
Comment (tpatpa) [inline docs/index.md]: Can add '(alphabetically)' at the end of the sentence here

File: readthedocs.yml [hunk 8]
```diff
@@ -1,14 +1,13 @@
 version: 2
 
+mkdocs:
```
Comment (tpatpa) [inline readthedocs.yml]: If you build in commands using `python -m mkdocs build --clean --site-dir $READTHEDOCS_OUTPUT/html --config-file mkdocs.yml`, is this section still necessary?

**Review by tpatpa — COMMENTED** [review]
Great work, @holmesclaire !!

I have added comments throughout that I think are worth considering.

File: readthedocs.yml (hunk 8 above)
Comment (holmesclaire) [reply readthedocs.yml]: just tested--not necessary

File: mkdocs.yml [hunk 9]
```diff
@@ -1,5 +1,6 @@
 site_name: HED tagging NSD images
 theme: readthedocs
+extra_css: [extra.css]
 nav:
 - 'About' : 'index.md'
 - 'Images and Annotations': 'nsd_images_annotations.md'
```
Comment (tpatpa) [inline mkdocs.yml]: Change here as well to 'Images and HED Annotations'

File: docs/generate_docs.py [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+    # load tsv of image identifiers and annotations
+    shared1000 = pandas.read_table('shared1000_HED.tsv')
+
+    # start markdown file
+    mdFile = MdUtils(file_name='nsd_images_annotations', title='Images and HED Annotations', author= 'Claire Holmes')
+
+    # set up headers for table
+    mdFile.new_line('The images on this webpage have been downsampled from [425, 425] to [200, 200] for upload to GitHub and this webpage.\n')
+    mdFile.new_line('**Only the first 4 images have been uploaded to GitHub and displayed here as an example. All images will be uploaded pending input from collaborators. All links to COCO images should be working.** \n note: actual NSD images may be a cropped version of the COCO image. The cropped version is the one displayed here and on GitHub.\n')
```
Comment (tpatpa) [inline docs/generate_docs.py]: Can this be seperated:

```
Only the first 4 images have been uploaded to GitHub and displayed here as an example. All images will be uploaded pending input from collaborators.

All links to COCO images should be working. Note: actual NSD images may be a cropped version of the COCO image. The cropped version is the one displayed here and on GitHub.
```

**Review by tpatpa — APPROVED** [review]
Thank you for quicky making the changes.
Two more minor edits, otherwise, lgtm!
IMO, this can be merged.

**Pull request merged**